content/guides/security/cors.md

```diff
@@ -163,3 +163,21 @@ following.
    from an incognito window, or from a whole other (not company owned) device.
 
 [mdn_cors]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
+
+### `Access-Control-Allow-Origin` header issues
+
+If you encounter issues with the `Access-Control-Allow-Origin` header, do the following:
+
+1. **Check if your domain is on the list of allowed origins** - Go to 
```

Comment:
Is it in bold on purpose? :) Also, I was thinking that this entry can be broken down into steps. 

- Check if your domain [...] 
   1. Go to the developer console...
   2. Click on the tab ...
   3. Add the domain ..

If both entries are not consecutive steps, maybe we can use bullet points 🤔 
* Check if your domain ...
* Check if your server ...